net-misc/kea/kea-2.6.3.ebuild

```diff
@@ -0,0 +1,242 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..14} )
+inherit autotools eapi9-ver flag-o-matic python-r1 systemd tmpfiles
+
+DESCRIPTION="High-performance production grade DHCPv4 & DHCPv6 server"
+HOMEPAGE="https://www.isc.org/kea/"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.isc.org/isc-projects/kea.git"
+else
+	SRC_URI="https://downloads.isc.org/isc/kea/${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
+
+LICENSE="MPL-2.0"
+SLOT="0"
+IUSE="debug +doc mysql +openssl postgres shell test"
+
+REQUIRED_USE="shell? ( ${PYTHON_REQUIRED_USE} )"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
+	dev-libs/boost:=
+	dev-libs/log4cplus:=
+	mysql? ( dev-db/mysql-connector-c:= )
+	!openssl? ( dev-libs/botan:2=[boost] )
+	openssl? ( dev-libs/openssl:0= )
+	postgres? ( dev-db/postgresql:* )
+	shell? ( ${PYTHON_DEPS} )
+"
+DEPEND="${COMMON_DEPEND}
+	test? ( dev-cpp/gtest )
+"
+RDEPEND="${COMMON_DEPEND}
+	acct-group/dhcp
+	acct-user/dhcp
+"
+BDEPEND="
+	doc? (
+		$(python_gen_any_dep '
+			dev-python/sphinx[${PYTHON_USEDEP}]
+			dev-python/sphinx-rtd-theme[${PYTHON_USEDEP}]
+		')
+	)
+	virtual/pkgconfig
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-2.2.0-openssl-version.patch
+)
+
+python_check_deps() {
+	use doc || return 0;
+	python_has_version "dev-python/sphinx[${PYTHON_USEDEP}]" \
+		"dev-python/sphinx-rtd-theme[${PYTHON_USEDEP}]"
+}
+
+pkg_setup() {
+	if use doc || use shell; then
+		python_setup
+	fi
+}
+
+src_prepare() {
+	default
+
+	# set shebang before autotools
+	if use shell; then
+		sed -e 's:^#!@PYTHON@:#!/usr/bin/env python3:' \
+			-i src/bin/shell/kea-shell.in || die
+	fi
+
+	# fix gtest detection
+	sed -e "s:dir/lib/:dir/$(get_libdir)/:" \
```

Comment:
I'd ask about improving this, but master uses meson, so whatever.

Reply:
I could revert and use the modified ax_gtest.m4 in tree if you prefer. Both do the job.

Reply:
Let's leave it as-is.